app/webpacker/components/Tickets/index.jsx

```diff
@@ -51,12 +53,25 @@ function TicketContent({ ticketDetails, currentStakeholder, sync }) {
         sync={sync}
         currentStakeholder={currentStakeholder}
       />
+      <TicketComments
+        ticketId={ticketDetails.ticket.id}
+        currentStakeholder={currentStakeholder}
+      />
       <TicketLogs logs={ticketDetails.ticket.ticket_logs} />
     </>
   );
 }
 
-export default function Tickets({ id }) {
+export default function Wrapper(args) {
+  return (
+    <WCAQueryClientProvider>
+      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
+      <Tickets {...args} />
```

Comment:
Why are you not spreading out the args directly in the definition of `Wrapper`, like any other React component would?

Reply:
Rule of thumb: If `eslint` complains about it (which in this case you can see by the fact that you had to introduce `eslint-disable-next-line` comment) then there's a good reason why it complains.
In 9 out of 10 cases, you should follow whatever ESLint tells you instead of suppressing the warning.